### DISCUSSION TITLE
Logo Discussion

### DISCUSSION THREAD

**Discussion by egekorkan** [w3c-cg/wot-cg#7] [General]
For a possible CG-oriented website, what should be our logo?

We can reuse the official logos after getting permission or create something ourselves. Some examples:
- WoT Japanese CG is using the WG/IG logo at https://wot-jp-cg.netlify.app/
- https://webofthings.org/ is using its own logo and is not officially tied to W3C
- https://www.w3.org/WoT/ This is the official website of WoT IG/WG

When creating one ourselves, we can design something completely new or tweak the colors a bit.

Based on the discussion outcomes, a vote can take place

**Comment by egekorkan** [comment]
A discussion on this came up during the W3C TPAC. We are using the WoT logo that is found at https://www.w3.org/WoT/IG/wiki/Web_of_Things_Logos but there were comments saying that the CG should use a different one. I would say that the CG can have its own logo but:
- It should be clear that it is used for the CG and not for the WoT as a whole.
- It should be somewhat aligned with the WoT logo. @koalie do you think the WoT CG can have an exception regarding the rule "The logo must stand alone: it cannot be combined with any other design element such as photography, type, borders, nor can it be incorporated into another logo."?

**Comment by koalie** [comment]
> A discussion on this came up during the W3C TPAC. We are using the WoT logo that is found at https://www.w3.org/WoT/IG/wiki/Web_of_Things_Logos but there were comments saying that the CG should use a different one. I would say that the CG can have its own logo but:
> 
>     * It should be clear that it is used for the CG and not for the WoT as a whole.
>     * It should be somewhat aligned with the WoT logo. @koalie do you think the WoT CG can have an exception regarding the rule "The logo must stand alone: it cannot be combined with any other design element such as photography, type, borders, nor can it be incorporated into another logo."?

It would be my preference that the CG doe not use a separate logo, or any logo whatsoever.
There is already confusion (not that particular CG, but all CGs) with regard to the output of CGs compared to WGs or IGs, that I feel that using a logo would further this type of confusion.

If the WoT CG is set on using a logo, I think it should use the available WoT one.

**Comment by egekorkan** [comment]
Thanks @koalie, this helps quite a lot. The question came after showing the WoT video at TPAC (see [here](https://www.w3.org/2023/09/TPAC/demos/what-is-wot.html)). I have also realized that the WoT logo is not a logo for WG or IG but it is a logo for WoT. As long as we are not using the WoT logo to point to CG, we should be fine and also aligned with not using a logo for CG.
